src/components/CardList.js

```diff
@@ -0,0 +1,51 @@
+import "./CardList.css";
+
+export default function CardList({ url, createdAt, desc, imgUrl }) {
+  const linkUrl = () => {
+    window.open(url);
+  };
+
+  const formattedDate = new Date(createdAt)
+    .toLocaleString("ko-KR", {
+      year: "numeric",
+      month: "2-digit",
+      day: "2-digit",
+    })
+    .replace(/\./g, ". ");
+
+  const timeAgo = (createdAt) => {
+    const currentTime = new Date();
+    const createdTime = new Date(createdAt);
```

Comment:
timeAgo 함수는 날짜를 매개변수로 받아서, 현재 시간을 기준으로 시간이 얼마나 흘렀는지에 대해 문자열 값을 리턴해주는 함수인데요~ 
createdTime이라는 변수명은 현재 카드 페이지에서만 해당되는 표현이라고 볼 수 있어요! 
만약에 관리자 페이지에서 회원탈퇴를 언제 했는지를 보여주는 페이지가 있다고 가정하면 여기서는 deletedAt과 같이 계정을 삭제한 시간이 현재 시간을 기준으로 얼마나 흘렀는지 확인할때도 timeAgo 함수를 사용할 수 있는데요~ 이렇게 사용하게 되면 18번 라인에 있는 createdTime은 말이 안되는 상황이 생길 것 같아요. 예를들면 이렇게요!
```js
const createdTime = new Date(deletedAt);
```

그래서 지금은 매개변수명이랑, createdTime이라는 이름을 조금 더 범용적으로 표현해주는게 필요하다고 느껴지는데요!
이런 느낌으로 해보면 어떨까 싶어요!
```suggestion
  const timeAgo = (time) => {
    const currentTime = new Date();
    const prevTime = new Date(time);
```